feat: Refactor docs, add new images and instructions (#140)
Project-Babble/BabbleDocs: docs/babbleofficaltracker/calibrationguide.mdx

![SlidersExample](@site/static/img/CalibrationGuideImages/BlankSlider.png)

The slider also contains a red dot visualizing the current state of the given blendshapes that is being detected.

![SliderRedDot](@site/static/img/CalibrationGuideImages/RedDotSlider.png)

## Adjusting values

The slider for each blendshape is controlled by **two handles** on either side of the blue bar. The left side is the _minimum_ cutoff, and the right side is the _maximum_. By tweaking those, you influence how the tracker picks up your expressions.&#x20;

The **red dot** indicates the current detected value of a given shape.&#x20;

:::info
The red dot will always move between the **far left and far right** side of the sliders and it's **not bound** to the position of the blue handles.&#x20;
You can check if any given shape is being detected by simply observing if the red dot is moving.
:::

#### **Left slider - minimum:** controls how quickly the tracker will pick up your movement. By moving it, you will change how big of a movement you have to make for it to be detected.

![SliderLeftSide](@site/static/img/CalibrationGuideImages/LeftSideSlider.png)

:::note

> This is not a scalar, anything below selected minimum value will be completely cutoff.

Author: I think we should rephrase this - instead of "scalar", more like:

```suggestion
This is not a gradual scale. Any value below the selected minimum will be completely excluded.
```
